package.json

```diff
@@ -110,6 +110,9 @@
     "remove-importmeta": "sed -i 's/import.meta.url/window.location.origin/g' \"./rust/kcl-wasm-lib/pkg/kcl_wasm_lib.js\"; sed -i '' 's/import.meta.url/window.location.origin/g' \"./rust/kcl-wasm-lib/pkg/kcl_wasm_lib.js\" || echo \"sed for both mac and linux\"",
     "lint-fix": "eslint --fix --ext .ts --ext .tsx src e2e packages/codemirror-lsp-client/src rust/kcl-language-server/client/src",
     "lint": "eslint --max-warnings 0 --ext .ts --ext .tsx src e2e packages/codemirror-lsp-client/src rust/kcl-language-server/client/src",
+    "url-checker":"./scripts/url-checker.sh",
+    "url-checker:overwrite":"npm run url-checker | sed '$d' > known-urls.txt",
+    "url-checker:diff":"./scripts/diff-url-checker.sh",
```

Comment:
Kind of a nitpick, so feel free to ignore, but most of these scripts are verbs so maybe `check-urls:` would be more consistent?

Reply:
```
    "url-checker":"./scripts/url-checker.sh",
    "url-checker:overwrite":"npm run url-checker | sed '$d' > known-urls.txt",
    "url-checker:diff":"./scripts/diff-url-checker.sh",
    "circular-deps": "dpdm --no-warning --no-tree -T --skip-dynamic-imports=circular src/index.tsx",
    "circular-deps:overwrite": "npm run circular-deps | sed '$d' | grep -v '^npm run' > known-circular.txt",
    "circular-deps:diff": "./scripts/diff-circular-deps.sh",
    "circular-deps:diff:nodejs": "npm run circular-deps:diff || node ./scripts/diff.js",
```

I followed the same pattern Jess implemented for circular deps. So the verbage is different